pages/common/keep-header.md

```diff
@@ -0,0 +1,20 @@
+# keep-header
+
+> Convenience utility that runs Unix commands in a header-aware fashion.
+> More information: <https://github.com/eBay/tsv-utils#keep-header>.
+
+- Pass all but the first line to command and keep the first line:
+
+`keep-header {{path/to/file}} -- {{command}}`
+
+- Sort files except for retaining a single header line:
+
+`keep-header {{path/to/file}} -- sort`
+
+- Read from stdin, sorting all except the first line:
+
```

Comment:
is the header the same as the first line?

Reply:
Yes it's two names for the same thing